components/buttons/buttons.css

```diff
@@ -25,38 +25,52 @@
 }
 
 .primary-btn-outlined{
-    background-color: var(--color-white);
-    color: var(--primary-color);
     border: 0.2rem solid var(--primary-color);
 }
 
-.secondary-btn-outlined{
+.primary-btn-outlined,
+.primary-btn-link,
+.primary-btn-icon{
     background-color: var(--color-white);
-    border: 0.2rem solid var(--color-gray);
-    color: var(--color-gray);
+    color: var(--primary-color);
 }
 
 .primary-btn-link{
-    background-color: var(--color-white);
-    color: var(--primary-color);
     border: none;
     margin:0.5rem 0.5rem;
     padding:0.5rem 0.5rem;
 }
 
-.secondary-btn-link{
-    background-color: var(--color-white);
-    color: var(--color-gray);
-    border: none;
+.primary-btn-float,
+.primary-btn-icon{
+    width:auto;
+    border:none;
 }
 
 .primary-btn-float{
-    width:auto;
-    border:none;
     padding:0.5rem 0.5rem;
     margin:0.75rem 0.75rem;
 }
 
+.primary-btn-icon{
+    margin:0.5rem 0.5rem;
+    padding: 0 0;
```

Comment:
here you can have margin:0.5rem, padding: 0

Reply:
code change done.